yubikey-luks-open

```diff
@@ -51,12 +50,12 @@ else
 fi
 
 if [ "$HASH" = "1" ]; then
-    P1=$(printf %s "$P1" | sha256sum | awk '{print $1}')
-    if [ "$DBG" = "1" ]; then echo "Password hash: $P1"; fi
+	P1=$(printf %s "$P1" | sha256sum | awk '{print $1}')
+		if [ "$DBG" = "1" ]; then echo "Password hash: $P1"; fi
 fi
 
-R="$(printf %s "$P1" | ykchalresp -"$YUBIKEY_LUKS_SLOT" -i- 2>/dev/null || true)"
-if [ "$DBG" = "1" ]; then echo "Yubikey response: $R"; fi
+R="$(printf %s "$P1" | ykchalresp -2 -i- 2>/dev/null || true)"
```

Comment:
Hardcoding slot 2 is rather poor fix that will render all keys with slot 1 unusable. Also formatting change is usually bad idea. It makes actual changes pretty unreadable and breaks commit history.

Reply:
Sorry, pushed change based on wrong version of file - one I got from debian repo and not the one from github rep (I know that "It was late in the night, and I was over the beer or two" is not an excuse :P ).  Now I fix it.